vpr/src/base/vpr_constraints_reader.cpp

```diff
@@ -10,31 +10,59 @@
 
 #include <fstream>
 #include "vpr_constraints_reader.h"
+#include "vtr_token.h"
 
-void load_vpr_constraints_file(const char* read_vpr_constraints_name) {
-    vtr::ScopedStartFinishTimer timer("Loading VPR constraints file");
+void load_vpr_constraints_files(const char* read_vpr_constraints_name) {
+    vtr::ScopedStartFinishTimer timer("Loading VPR constraints file(s)");
 
     VprConstraintsSerializer reader;
 
-    if (vtr::check_file_name_extension(read_vpr_constraints_name, ".xml")) {
-        try {
-            std::ifstream file(read_vpr_constraints_name);
-            void* context;
-            uxsd::load_vpr_constraints_xml(reader, context, read_vpr_constraints_name, file);
-        } catch (pugiutil::XmlError& e) {
-            vpr_throw(VPR_ERROR_ROUTE, read_vpr_constraints_name, e.line(), "%s", e.what());
+    // file name from arguments could be a serial of files, seperated by colon ":"
```

Comment:
You may need to update the vpr documentation about command-line options. This is a new feature.

Current documentation:
https://docs.verilogtorouting.org/en/latest/vpr/command_line_usage/#cmdoption-vpr-read_vpr_constraints